### PR TITLE
Feature/itl 7 create the db structure

### PR DESCRIPTION
**Pull request by aronnorbert24**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@vueuse/core": "^10.3.0",
+    "mongoose": "^7.5.0",
```
Comment (tamasm-wg) [inline client/package.json]: do you need mongoose on the client side ?

File: server/src/models/Todo.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import mongoose from "mongoose";
+
```
Comment (SzaszWebgurus) [inline server/src/models/Todo.ts:2]: add an interface as well for each schema

File: server/src/models/Todo.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	},
+
+	userId: {
+		type: mongoose.Schema.Types.ObjectId,
+		ref: "User",
+		required: true,
+	},
+});
+
+module.exports = mongoose.model("Todo", todoSchema);
```
Comment (SzaszWebgurus) [inline server/src/models/Todo.ts]: todo, with lowercase

File: server/src/models/User.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+		required: true,
+	},
+
+	email: {
+		type: String,
+		required: true,
+	},
+});
+
+module.exports = mongoose.model("User", userSchema);
```
Comment (SzaszWebgurus) [inline server/src/models/User.ts]: user, with lowercase

**Review by SzaszWebgurus — CHANGES_REQUESTED** [review]

**Pull request merged**